teams/vsp/teams/product-dev-support/product-dev-support-team-charter.md

```diff
@@ -1,35 +1,39 @@
-# [Team Name] Charter
+# Product support Charter
 
 ---
 
 ## Why we're here
 
-### North Star:
-_Lorem ipsum_
-
 ### Product Suite (if applicable): 
-_i.e. Health Care, Education, Global UX, Authenticated Experience, etc - ask Chris J or your VA Product Lead if you're unsure!_
+Onboarding, long-term support of VFS teams
 
 ### Team Mission Statement:
-_Lorem ipsum_
+Provide expert design, engineering, and product guidance to VFS teams to ensure their applications are consistent, reliable, and maintainable as well as embody the culture of human-centeredness. 
 
-### Product Portfolio:
-_List of products within your team's perview - make sure the list of products here aligns to what's captured in the [List of VA.gov Products doc](https://docs.google.com/spreadsheets/d/1hzz6whEGoQJQbiNvIggirhydYYdv57nfOZfLvFqZ1pQ/edit)_
 
 ### Responsibilities:
-_High level problem space and user goals for this team_
+- Own + maintain VSP Service Blueprint overall
```

Comment:
Is there a link you can add to the blueprint? I'm happy to help with formatting if you need it